util/feeding.py

```diff
@@ -42,9 +43,12 @@ def samples_to_mfccs(samples, sample_rate):
 
 
 def audiofile_to_features(wav_filename):
+    global is_pertub
     samples = tf.io.read_file(wav_filename)
     decoded = contrib_audio.decode_wav(samples, desired_channels=1)
-    features, features_len = samples_to_mfccs(decoded.audio, decoded.sample_rate)
+    if is_pertub:
+        samples = pertub(decoded.audio)
```

Comment:
you're passing decoded audio to perturb and then perturb is trying to access 'audio' and 'sample_rate'attribute from there. Decoded audio is a numpy array. What's goin on here?

Also, if is_perturb==False, then shouldn't you be passing decoded.audio to samples_to_mfcc?